### ISSUE TITLE
Return browsable gt object in Python repl

### ISSUE THREAD

**Issue by jooyoungseo** [posit-dev/great-tables#100]
Thanks for this great package! I am suggesting one feature request below.

In R {gt} package, `gt()` object returns browsable table in user's default browser. This is very convenient way for screen reader users to navigate data table in an alternative way. Could you please consider this in Python package as well?

Currently, `GT()` method returns html code in Python repl. Browsable GT object might be more useful in an interactive terminal from end-user's perspective.

**Comment by machow** [comment]
Thanks for looking into this issue! I've been looking around for python libraries that do something similar, and it seems like plotly might be a good example of this:

```python
import plotly.express as px

fig = px.scatter(x=[0, 1, 2, 3, 4], y=[0, 1, 4, 9, 16])

# opens a browser window
fig.show()
```

Since the gt for R opens a browser aimed at a temporary .html file, we should be able to do the same. WDYT of that approach? In python, I wonder if we should add a method, like `.preview()`, that triggers this behavior.

I think the `.__repr__` method triggering a browser window to open might be surprising to python users but a `.preview()`. For plotly, it looks like they do trigger a browser window when you dump to console, but only for IPython, not for the python repl (there, you have to use `fig.show()`)...

**Comment by jooyoungseo** [comment]
Maybe, I could be wrong, but in R, {gt} seems to rely on {htmltools} and `gt::gt()` appears to return `htmltools::browsable()` object.

I created a feature request https://github.com/posit-dev/py-htmltools/issues/79 to ask if {py-htmltools} could possibly support `htmltools.browsable()` methods. It would be great if {great_tables} depends on {py-htmltools} once it gets more reliable.

**Comment by jooyoungseo** [comment]
@machow -- I have just learned from @cpsievert that we could use `show()` methods against {py-htmltools} object. What if {great_tables} depends on {py-htmltools} for this feature and beyond?

**Comment by machow** [comment]
Great Tables uses htmltools, so this is perfect! I was able to get it to appear in the browser using this code in the IPython console...

```python
from great_tables import GT, exibble
from htmltools import HTML, TagList

res = GT(exibble).render("html")
el = HTML(res)

TagList(el).show("browser")
```

Note that I had to pass "browser" as an argument. The default argument of "auto" didn't seem to do anything.

It seems like at the very least we should add a `.show()` or `.preview()` method, which holds the logic for viewing the table anywhere (jupyterlab, IPython console, etc..). This could use something similar to the code above.
